### ISSUE TITLE
Focus Returned to Captions Menu instead of Caption Settings Menu Item when closing Caption Settings

### ISSUE THREAD

**Issue by josephshanak** [videojs/video.js#6949]
## Description
During an accessibility audit of a project I work on that uses VideoJS we received the following feedback about how the focus is handled after closing the Caption Settings menu.

> Users will expect to be returned to the item that activated the dialog when they close the dialog.
> When a user closes the dialog, focus should be moved back to the menu item for "caption settings".

It seems like the behavior expected should have been implemented as part of #2746

> Keyboard focus does not return to the control which opened the dialog when it is closed

## Steps to reproduce
1. Go to https://videojs.com/advanced/?video=elephantsdream
2. Tab to Captions menu
3. Enter Caption Settings
4. Tab to Close button and activate

## Results
Focus is returned to Captions Menu on the control bar

### Expected
Focus is returned to "Caption Settings" menu item.

### Actual
Focus is returned to Captions Menu on the control bar

### Error output
N/A

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
Happens on 7.7.5 and 7.10.2 at least

#### browsers
Firefox, Chrome. Possibly others but untested.

#### OSes
Verified on MacBook Pro running MacOS Catalina 10.15.7

### plugins
Unsure

**Comment by gkatsev** [comment]
I think we return the captions to the menu because the menu ends up getting closed. It does seem like maybe the menu should get re-opened and focus brought back into the menu.
@OwenEdwards any thoughts on this?

**Comment by kannapples** [comment]
I created a PR for this issue. The fix I came up with isn't as modular as I'd like it to be - its specific to the caption settings modal. This means if another modal is added, it wont automatically include this accessibility fix. I couldn't figure out a way to handle this gracefully, but I'm happy to make updates if you have any suggestions.

**Comment by OwenEdwards** [comment]
@gkatsev there is some disagreement in accessibility circles about the expected behavior in this situation. I'm getting some clarification, but I don't think that #6971 should be merged until I have a more definite answer (also, #6971 isn't robust for different cases when other menus can open the Captions Settings Dialog)

**Comment by kannapples** [comment]
@OwenEdwards were you able to dig up any clarification about the expected behavior here?

**Comment by OwenEdwards** [comment]
Confirmed that this is acceptable behavior for focus movement when closing the Captions Setting Dialog.
